ci/dev/watch.ts

```diff
@@ -173,30 +165,27 @@ class Watcher {
           console.log("[plugin]", original)
         }
         if (line.includes("Watching for file changes")) {
-          bundle.then(restartServer)
+          bundleBrowserCode(browserFiles)
```

Comment:
We can remove this since this is a `tsc` for plugins which won't recompile the browser files.